kuma/landing/jinja2/landing/homepage.html

```diff
@@ -8,6 +8,10 @@
   {{ super() }}
 
   {% stylesheet 'home' %}
+
+  {% if waffle.flag('developer_needs') %}
```

Comment:
If I understand correctly,  you're not testing contrib_beta here because that is a special case and its styles are included elsewhere. But when other banners are added in the future, this line will have to be updated to check for them.  (I suppose it might make sense at some point to have a global waffle flag that just controls whether any banners are active, and then you could test that instead of listing all possible banners.)  But this is fine for now.

Reply:
Yes, we will have eventually have a single flag for this.